src/rgw/rgw_admin.cc

```diff
@@ -5494,6 +5494,15 @@ int main(int argc, const char **argv)
   }
 
   if (opt_cmd == OPT_OBJECT_RM) {
+    if (bucket_name.empty()) {
+      cerr << "ERROR: bucket not specified" << std::endl;
+      return EINVAL;
+    }
+    if (object.empty()) {
+      cerr << "ERROR: object not specified" << std::endl;
+      return EINVAL;
+    }
+
```

Comment:
its probably a good idea to pull these up into a new if block above to avoid the duplication. what do you think?
```c++
if (opt_cmd == OPT_OBJECT_RM || opt_cmd == OPT_OBJECT_UNLINK || opt_cmd == OPT_OBJECT_STAT) {
   ...
```